index.js

```diff
@@ -36,14 +45,7 @@ const displayBook = () => {
   });
 };
 
-document.querySelector('.btn-submit').addEventListener('click', (e) => {
-  e.preventDefault();
-  addBook();
-  displayBook();
-  document.querySelector('#title').value = '';
-  document.querySelector('#author').value = '';
-});
-
+/* Remove Book */
 const deleteBook = (elem) => {
   const books = JSON.parse(localStorage.getItem('books'));
   if (books.length === 1) {
```

Comment:
- [ ] Same issue as above. Please ensure that the `deleteBook` method is moved into the class to make it a class method.